src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Publisher.kt

```diff
@@ -56,4 +52,15 @@ class Publisher(private val topicName: String) {
             publisher?.shutdown()
         }
     }
+
+    companion object {
```

Comment:
You actually don't have to wrap functions into companion objects. Functions can happily sit as top-level objects in a file and can be freely used from other files.

Reply:
Done